backend/Users/Users.ts

```diff
@@ -71,3 +68,18 @@ export async function findUser(email, password) {
     return null;
   }
 }
+
+export async function findUserByEmail(
+  email: string
+): Promise<UserModel | null> {
```

Comment:
I think the UserModel might be the type for sequelize.models.User rather than for a single User object. If so, this is the wrong type and it should be User rather than UserModel. I am not as familiar with sequelize types so I don't know which one is correct. The way I'd verify is I'd hover over UserModel in Visual Studio Code and see if it includes functions like findAll() or if it's just the properties of the Ueer object. If hovering doesn't clarify I would see what properties are available on it by starting the auto completion on the returned object to see if it includes functions or not. It should not have any functions on it.  Additionally I would look into whether sequelize's findWhere function supports generics and would implement them here if they do to ensure further type safety. However, it's okay if you skip the generics part because that's probably new for you, but we do need to verify that the return type should be either User or UserModel

Reply:
I have confirmed that there are no functions attached to UserModel type.